CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+## v1.3.0 (04/03/2020)
+
+**Common**
+
+- Updated dependencies.
+- Fixed bugs:
+- - Version "0" is incorrectly handled
```

Comment:
```suggestion
  - Version "0" was incorrectly handled. This is now fixed.
```
Means two spaces and a dash is a sub-list. You should not use a double dash.
Please also place a dot after every sentence.

Reply:
makes sense, updated. Thank you